plugins/intel-me/README.md

```diff
@@ -0,0 +1,19 @@
+# Intel ME
```

Comment:
Doesn't Intel call this CSME these days?

Reply:
As I understand it, CSME is a type of ME, and there are a few other types too.